tayoto_mikaellaantonette/script.js

```diff
@@ -0,0 +1,135 @@
+const form = document.querySelector("form");
+const nameInput = document.querySelector("#name");
+const commentInput = document.querySelector("#gcomment");
+const commentList = document.querySelector("#comment_list");
+const sortAscButton = document.querySelector("#sort_asc");
+const sortDescButton = document.querySelector("#sort_desc");
+
+form.addEventListener("submit", function (event) {
+  event.preventDefault();
+  const name = nameInput.value.trim();
+  const comment = commentInput.value.trim();
+  const date = new Date();
+  if (!name || !comment) {
+    return;
+  }
+
+  const newComment = document.createElement("li");
+  newComment.innerHTML = `<strong>${name}</strong> - ${comment} - 
+    <time datetime="${date}">${date.toLocaleString()}</time>`;
+
+  const listItem = document.createElement("li");
+  listItem.appendChild(newComment);
+  commentList.appendChild(listItem);
+  nameInput.value = "";
+  commentInput.value = "";
+  form.querySelector('button[type="submit"]').disabled = false;
+});
+
+function sortComments(ascending) {
+  const comments = Array.from(commentList.children);
+  comments.sort(function (a, b) {
+    const aDate = new Date(a.querySelector("time").getAttribute("datetime"));
+    const bDate = new Date(b.querySelector("time").getAttribute("datetime"));
+    return ascending ? aDate - bDate : bDate - aDate;
+  });
+
+  commentList.innerHTML = "";
+  comments.forEach(function (comment) {
+    commentList.appendChild(comment);
+  });
+}
+
+sortAscButton.addEventListener("click", function () {
+  sortComments(true);
+});
+
+sortDescButton.addEventListener("click", function () {
+  sortComments(false);
+});
+
+function searchCountry() {
+  var countryName = document.getElementById("country_input").value.trim();
```

Comment:
Use let instead of var
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d183194fb_0_0